conans/client/graph/proxy.py

```diff
@@ -92,6 +92,18 @@ def _get_recipe(self, reference, remotes, update, check_update):
                 status = RECIPE_INCACHE_DATE_UPDATED
         return recipe_layout, status, remote
 
+    @staticmethod
+    def _should_update(reference, update):
```

Comment:
Probably just make it a free function here in ``proxy.py`` at this moment, and import it from ``range_resolver``